sdk/daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -2514,21 +2569,24 @@ private[lf] object SBuiltin {
       templateId: Identifier,
       templateArg: SValue,
       keyOpt: SValue,
+      allowCatchingContractInfoErrors: Boolean,
```

Comment:
I am probably too picky here, but would not be better to have the inverse flag i.e. `preventCatchingErrors` ?

Reply:
I've considered it, but I was remembered of some advice I read somewhere that said it's better not to have flags with negative names because then in case of "false" the reader has to do a double-negation simplification in their mind. `preventCatchingErrors = false` -> allows the catching of errors.